app/src/main/java/org/fossasia/pslab/fragment/ControlFragmentAdvanced.java

```diff
@@ -83,6 +85,85 @@ public void onClick(View v) {
         etwidgetControlAdvanced10.init(getContext(), 0.1, 0.0, 1.0);
         etwidgetControlAdvanced11.init(getContext(), 1.0, 0.0, 360.0);
 
+        buttonControlAdvanced1.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
```

Comment:
I changed the string comparison part. Are you sure of `NumberFormatRestriction`, we already restricted the text to numbers and moreover Android Studio itself highlights the exceptions like these if they occur?